pyproject.toml

```diff
@@ -12,7 +12,7 @@ keywords = [
   "uv", "requirements", "packaging"
 ]
 classifiers = [
-  "Development Status :: 4 - Beta",
+  "Development Status :: 4 - Stable",
```

Comment:
I think this needs to be "Development Status :: 5 - Production/Stable"

Reply:
Thanks, I should have looked that up.

Reply:
For reference https://pypi.org/classifiers/

Reply:
Maybe overkill, but https://pypi.org/project/validate-pyproject/ can validate a `pyproject.toml`, including classifiers. It would have indeed detected the issue here:

```console
$ uvx --from 'validate-pyproject[all]' validate-pyproject pyproject.toml
Invalid file: pyproject.toml
[ERROR] `project.classifiers[0]` must be trove-classifier
```

There's even a `pre-commit` hook, so maybe that can be run locally only when `pyproject.toml` is updated, and every time on the CI as it should be quick to run? Happy to add it if you think it's worth doing.

Reply:
I don't really feel the need for a pre-commit hook but a CI check seems nice to have!